### PR TITLE
new: Add `fzf` plugin.

### PR DESCRIPTION
**Pull request by suin**

### CODE REVIEW

File: registry/data/third-party.json [hunk 1]
```diff
@@ -162,6 +149,19 @@
...
+      "format": "toml",
+      "name": "CUE",
+      "description": "CUE makes it easy to validate data, write schemas, and ensure configurations align with policies.",
+      "author": "appthrust",
+      "homepageUrl": "https://github.com/appthrust/proto-toml-plugins",
+      "repositoryUrl": "https://github.com/appthrust/proto-toml-plugins/tree/main/cue",
+      "bins": [
+        "cue"
+      ]
+    },
```
Comment (suin) [inline registry/data/third-party.json:164]: **Note:** This change is not related to the main purpose of this PR. It was automatically fixed by `just gen`.

**Comment by milesj** [comment]
Thanks!

The test failure is unrelated, so I'll merge through.

**Comment by suin** [comment]
Thank you for merging this 😊
